backend/package.json

```diff
@@ -6,21 +6,28 @@
   "scripts": {
     "serve": "nodemon -r esm src/index.js",
     "lint": "eslint --ext .js",
-    "test": "jest --verbose"
+    "test": "jest --verbose --runInBand",
+    "clean": "kill -9 $(lsof -t -i:4000)",
```

Comment:
haha I meant to delete it. It was just because we were having troubles with stopping the server and it restarting and crashing because the port was occupied somehow. will delete it

Reply:
ah, nice :grin: